src/data/perf_stat.rs

```diff
@@ -6,28 +6,33 @@ use crate::{PDError, PERFORMANCE_DATA, VISUALIZATION_DATA};
 use anyhow::Result;
 use chrono::prelude::*;
 use ctor::ctor;
-use log::{trace, warn};
+use log::{error, info, trace, warn};
 use perf_event::events::{Raw, Software};
 use perf_event::{Builder, Counter, Group, ReadFormat};
 use serde::{Deserialize, Serialize};
 use std::io::{BufRead, BufReader, ErrorKind};
 use std::sync::Mutex;
 
 #[cfg(target_arch = "aarch64")]
-use crate::data::grv_perf_events;
+pub mod arm64_perf_list {
+    pub static GRV_EVENTS: &[u8] = include_bytes!("grv_perf_list.json");
+}
 #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-use {
-    crate::data::{
-        amd_genoa_perf_events::GENOA_CTRS, amd_milan_perf_events::MILAN_CTRS, amd_perf_events,
-        intel_icelake_perf_events::ICX_CTRS, intel_perf_events,
-        intel_sapphire_rapids_perf_events::SPR_CTRS, utils::get_cpu_info,
-    },
-    indexmap::IndexMap,
-};
+pub mod x86_perf_list {
+    /// Intel+ events.
+    pub static INTEL_EVENTS: &[u8] = include_bytes!("intel_perf_list.json");
+    pub static ICX_CTRS: &[u8] = include_bytes!("intel_icelake_ctrs.json");
+    pub static SPR_CTRS: &[u8] = include_bytes!("intel_sapphire_rapids_ctrs.json");
+
+    /// AMD+ events.
+    pub static AMD_EVENTS: &[u8] = include_bytes!("amd_perf_list.json");
+    pub static GENOA_CTRS: &[u8] = include_bytes!("amd_genoa_ctrs.json");
+    pub static MILAN_CTRS: &[u8] = include_bytes!("amd_milan_ctrs.json");
+}
```

Comment:
Why the difference in name between "Events" and "Counters" here?  Is now a good time to get rid of that?

Reply:
Counters are specific to a generation. For ex: AMD Genoa is AMD_EVENTS with the GENOA_CTRS collated.